presidio-analyzer/Dockerfile.local

```diff
@@ -0,0 +1,30 @@
+ARG REGISTRY=presidio.azurecr.io
```

Comment:
Worth adding some documentation on the difference between the docker files (in the files themselves)

Reply:
👍 
is 'local' == 'standalone'?

Reply:
local is on a local dev environment. non-local is the one used in CI. added doc in files